com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -242,6 +242,7 @@
     public static final String SERVICE_BROKER_DOES_NOT_EXIST = "Service broker with name \"{0}\" does not exist";
     public static final String EXCEPTION_OCCURED_ERROR_MSG = "Exception occured during execution: \"{0}\"";
     public static final String EXECUTING_HOOK_0 = "Executing hook \"{0}\"";
+    public static final String POLLING_IN_PROGRESS_SERVICES = "There is still services in progress state. Polling service operations...";

```

Comment:
The output for creation and deletion of async services is the following:
0 of x done.
Should we display information here about how many are still in progress?
If you want to do it, try changing the message in order to be more understandable from user point of view.

Reply:
After this message it will start polling for service operations and it will print progress state in the same way as creation and deletion of services.